commit 6ef766dc7b016fff5003661e7cd5e65a1f43d607
Author: Renato Golin <rengolin@systemcall.eu>
Date:   Fri Aug 19 18:20:48 2022 +0100

    Fix MLIRTosaDialect ref in CMake

diff --git a/tosa/CMakeLists.txt b/tosa/CMakeLists.txt
index bf4611e1..46c501ce 100644
--- a/tosa/CMakeLists.txt
+++ b/tosa/CMakeLists.txt
@@ -33,7 +33,7 @@ set(LIBS
         ${conversion_libs}
         MLIROptLib
 
-        MLIRTosa
+        MLIRTosaDialect
         MhloRegisterDialects
         MhloRegisterDialects
 
